cst/plugins/eu.esdihumboldt.cst.functions.numeric/src/eu/esdihumboldt/cst/functions/numeric/sequentialid/SequentialIDConstants.java

```diff
@@ -45,10 +45,8 @@
 
 	/**
 	 * The start value for sequences.
-	 * 
-	 * TODO make this configurable?
 	 */
-	public static final int START_VALUE = 1;
+	public static final String START_VALUE = "startValue";
```

Comment:
As you changed the semantics maybe also change the name of the constant? e.g. to `PARAM_START_VALUE` following the pattern for the other function parameter names?